Makefile

```diff
@@ -1536,7 +1536,8 @@ pkg/sql/colexec/vec_comparators.eg.go: pkg/sql/colexec/vec_comparators_tmpl.go
 pkg/sql/colexec/window_peer_grouper.eg.go: pkg/sql/colexec/window_peer_grouper_tmpl.go
 
 $(EXECGEN_TARGETS): bin/execgen
-	execgen $@
+	execgen $@ > $@.tmp; cmp $@.tmp $@ && rm -f $@.tmp || mv $@.tmp $@
```

Comment:
Splitting it into two lines is more "idiomatic".

Reply:
Yeah but it's super noisy and gross in the Makefile output so I'll avoid unless you insist.